sphinx_gallery/gen_rst.py

```diff
@@ -275,7 +278,10 @@ def save_thumbnail(image_path_template, src_file, file_conf, gallery_conf):
         img = gallery_conf.get("default_thumb_file", img)
     else:
         return
-    scale_image(img, thumb_file, *gallery_conf["thumbnail_size"])
+    if ext == 'svg':
+        copyfile(img, thumb_file)
```

Comment:
for GIF, I wonder if there's an easy way to just pull out the first frame

Reply:
I would try to actually scale it properly, it's probably not much more effort (loop rather than take first) and would have a nice payoff. But this is for the next PR :)